app.py

```diff
@@ -4,16 +4,20 @@
 import user
 import profile
 import db
+from flask_login import LoginManager, login_required, login_user, logout_user, current_user
```

Comment:
ModuleNotFoundError: No module named 'flask_login'

Can you add the related package into `requirements.txt`?